src/settingsView.ts

```diff
@@ -312,6 +320,19 @@ function htmlForWebview(vscode: vscodeTypes.VSCode, extensionUri: vscodeTypes.Ur
           vscode.postMessage({ method: 'toggle', params: { setting: event.target.getAttribute('setting') } });
         });
       }
+
+      function updateEmbedTraceViewer() {
```

Comment:
Any reason this is not a part of updateSettings?

Reply:
That function hides the **Embed Trace Viewer** when the project is from an older version, but now that I look deeper into it, an alternative is to keep that option always visible but show a warning and fallback to the spawn process version if the project doesn't support embedded mode. That is in line with `showTrace` setting, that is always shown even when the project uses an old playwright version.